ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/dcatserver/services/DcatServerServiceImpl.java

```diff
@@ -1,21 +1,73 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatserver.services;
 
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatdataset.entities.DcatDataset;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatdataset.services.DcatDatasetService;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatserver.entities.DcatServer;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatserver.exceptions.MissingDcatServerException;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatserver.repositories.DcatServerRepository;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.exceptions.DcatAlreadyConfiguredException;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.DcatShaclValidator;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.view.entity.DcatView;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.view.service.DcatViewService;
 import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Statement;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
 
 @Service
 public class DcatServerServiceImpl implements DcatServerService {
+
 	private final DcatServerRepository dcatServerRepository;
+	private final DcatViewService dcatViewService;
+	private final DcatDatasetService dcatDatasetService;
+	private final String hostName;
+
+	private final DcatShaclValidator dcatShaclValidator;
 
-	public DcatServerServiceImpl(DcatServerRepository dcatServerRepository) {
+	public DcatServerServiceImpl(DcatServerRepository dcatServerRepository,
+			DcatViewService dcatViewService,
+			DcatDatasetService dcatDatasetService,
+			@Value("${ldes-server.host-name}") String hostName,
```

Comment:
Out of scope, but I'd like to get this out of the app config. So that this can still be changed on the fly :)